src/components/AboutModal/__snapshots__/AboutModal.test.js.snap

```diff
@@ -1,3 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`AboutModal renders correctly when hidden 1`] = `null`;
+
+exports[`AboutModal renders correctly when shown 1`] = `"<div role=\\"dialog\\"><div class=\\"modal-backdrop fade in\\"></div><div role=\\"dialog\\" tabindex=\\"-1\\" style=\\"display: block; padding-right: 0px;\\" class=\\"fade in modal\\"><div class=\\"modal-dialog\\"><div class=\\"modal-content about-modal-pf\\" role=\\"document\\"><div class=\\"modal-header\\"><button class=\\"close\\"><span aria-hidden=\\"true\\" class=\\"pficon pficon-close\\" title=\\"Close\\"></span><span class=\\"sr-only\\">Close</span></button></div><div class=\\"modal-body\\"><h1>Product Title</h1><div class=\\"product-versions-pf\\"><ul class=\\"list-unstyled\\"><li class=\\"\\"><strong>Label</strong> Version</li></ul></div><div class=\\"trademark-pf\\">Trademark and Copyright Information</div></div><div class=\\"modal-footer\\"><img src=\\"\\" alt=\\"Patternfly Logo\\"></div></div></div></div></div>"`;
```

Comment:
is there any way to clean up this snapshot? the way `component.html()` is snapping is not so pretty...

Reply:
When you say "not so pretty" and "clean up" I understand your meaning to be, that the single line escaped html output is difficult to read at a glance.  The formatting is shite, is that about right? 

Or is your issue with the type of output?  I would argue the content rendered by a component *far* more meaningful than alternatives what is rendered when prop `show = true`.

Reply:
Se how we use `toJson` in foreman:
https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/common/Alert/AlertBody.test.js#L11

Reply:
Oh but this isn't the shallow wrapper, its the mount wrapper, already html... we can `JSON.stringify()` to make remove all the escaping... (updated the pr to reflect)

Reply:
@sharvit greaaaaaaaaat suggestion!  while the wrapper type is different, `mount` vs `shallow` used in foreman... it looks like the `.render()` output yields the same result... will go back and fixup a few other instances in this pr on monday